wallet/wallet.go

```diff
@@ -214,12 +214,16 @@ func (w *Wallet) RequestMint(amount uint64) (*nut04.PostMintQuoteBolt11Response,
 		return nil, fmt.Errorf("error decoding bolt11 invoice: %v", err)
 	}
 
-	invoice := lightning.Invoice{
-		Id:             mintResponse.Quote,
-		PaymentRequest: mintResponse.Request,
-		PaymentHash:    bolt11.PaymentHash,
-		Amount:         amount,
-		Expiry:         mintResponse.Expiry,
+	invoice := storage.Invoice{
```

Comment:
Is it possible to pass a description or description hash when requesting a mint?

Reply:
Ok, no problem. Thanks!